File: Embassat/View/Info/CADEMInfoHeaderView.m
//
//  CADEMInfoHeaderView.m
//  Embassat
//
//  Created by Joan Romano on 13/04/14.
//  Copyright (c) 2014 Crows And Dogs. All rights reserved.
//

#import "CADEMInfoHeaderView.h"

@interface CADEMInfoHeaderView ()

@property (nonatomic, weak) IBOutlet UIImageView *coverImage;
@property (nonatomic, weak) IBOutlet UILabel *titleLabel;

@end

@implementation CADEMInfoHeaderView

- (void)awakeFromNib
{
    [super awakeFromNib];
    
    RAC(self.titleLabel, text) = RACObserve(self, title);
    RAC(self.coverImage, image) = RACObserve(self, cover);
    
    self.titleLabel.font = [UIFont em_titleFontOfSize:16.0f];
}

@end

File: Embassat/View/Info/CADEMInfoHeaderView.swift
//
//  CADEMInfoHeaderView.swift
//  Embassa't
//
//  Created by Joan Romano on 24/05/15.
//  Copyright (c) 2015 Crows And Dogs. All rights reserved.
//

import UIKit

public class CADEMInfoHeaderView: UICollectionReusableView {
    
    @IBOutlet weak var titleLabel: UILabel?
    @IBOutlet weak var coverImage: UIImageView?
    
    public var title: String = "" {
        didSet {
            titleLabel?.text = title
        }
    }
    public var cover: UIImage? = nil {
        didSet {
            coverImage?.image = cover
        }
    }
    
    public override func awakeFromNib() {
        super.awakeFromNib()
        
        titleLabel?.font = UIFont.detailFont(ofSize: 20.0)
    }
}
